assets/scss/app.scss

```diff
@@ -1,3 +1,12 @@
+@import "~bootstrap/dist/css/bootstrap.css";
+@import "~font-awesome/scss/font-awesome";
+@import "~admin-lte/dist/css/alt/AdminLTE-without-plugins.css";
+@import "~icheck/skins/square/blue.css";
+@import "../vendor/bootstrap-datetimepicker.css";
```

Comment:
You can use the `node_modules` version of this lib or do you want to do this in an other PR?

Reply:
We have a problem with that lib... The version 3, only have one version available on npm, and it is coming with less files, not sass. I am afraid that until we update to version 4, we can't use npm for that dependency.

Reply:
This is the version on npm: https://www.npmjs.com/package/eonasdan-bootstrap-datetimepicker/v/3.1.3

And it comes with: https://github.com/Eonasdan/tempus-dominus/tree/v3.1.3/src

For some reason, the library installed through npm does not have the compiled files... (that is a problem we have too with jquery-ui, but at least that lib uses pain css, so it can be handled later)

Reply:
I was trying to upgrade it but we have another problem with that. The dates form types are here: https://github.com/sonata-project/form-extensions/tree/1.x/src/Type

So it can't be done that easy... IMO we will end removing this dependency here and adding it to the form-extensions.

So... at the end, I think for now this is the best option, but will probably changge in the next PRs, at some point.

Reply:
Maybe adding a comment to explain why you use it like this could help the futur dev :)

Reply:
Done